api/src/pcapi/flask_app.py

```diff
@@ -140,6 +142,11 @@ def add_security_headers(response: flask.wrappers.Response) -> flask.wrappers.Re
         restrictions=profiling_restrictions,
     )
 
+if int(os.environ.get("ENABLE_RQ_PROMETHEUS_EXPORTER", "0")):
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏ça retourne `True` si on a pas la clé. C'est pas voulu je pense

```suggestion
if int(os.environ.get("ENABLE_RQ_PROMETHEUS_EXPORTER", False)):
```

<img width="118" alt="image" src="https://github.com/pass-culture/pass-culture-main/assets/14235661/ff3a53dc-2a67-4b21-8e89-38a1f3a2ef8c">

Reply:
J’ai utilisé `int(os.environ.get` plutôt que `bool(os.environ.get` qui est la façon utilisée dans le code

Reply:
edit: FAUX, tu cast en `int`, my bad 😅